test/node/dataset_partial_io_test.cpp

```diff
@@ -65,6 +66,22 @@ TEST_F(PartialIO, test_read_write_scalar_int)
 }
 
 
+TEST_F(PartialIO, test_read_write_points)
+{
+  dataspace::Simple space{{10}};
+  auto type = datatype::create<int>();
+
+  node::Dataset dset(root_,Path("data"),type,space);
+
+  std::vector<int> write {1,2,3,4,5};
+  dset.write(write, dataspace::Points({{1}, {2}, {3}, {4}, {5}}));
```

Comment:
Test is confusing, better to use values which don't match the indices they are written to.